src/development/tools/devtools/memory.md

```diff
@@ -101,66 +117,65 @@ This chart displays DevTools events (such as manual GC, VM GC,
 Snapshot, monitor Allocations **Track** and **Reset** of accumulators button
 clicks) in relation to the memory chart timeline. Clicking over the
 markers in the Event timeline displays a hover card of the time when
-the event occurred. This may help identify when a memory leak might have
-occurred in the timeline (x-axis).
+the event occurred. This might help identify when a memory leak has
+possibly occurred in the timeline (x-axis).
 
 ![Screenshot of the event timeline legend]({{site.url}}/assets/images/docs/tools/devtools/memory_eventtimeline_legend.png)
 
-This legend shows the symbol for each DevTools event and its meaning
+This legend shows the symbol for each DevTools event and its meaning:
 
 <dl markdown="1">
```

Comment:
I would like to consult whether we can have a better format here instead of `<dl>`, `<dt>`, and `<dd>`. The current format makes us hard to read and modify content.

Reply:
Oops. I totally forgot to share the staged link. (It won't show images, but should otherwise be good.)
https://sz-flutter.firebaseapp.com/development/tools/devtools/memory